gen/testdata/constants/constants.go

```diff
@@ -0,0 +1,51 @@
+// Code generated by thriftrw
+
+package constants
+
+import (
+	"github.com/thriftrw/thriftrw-go/gen/testdata/containers"
+	"github.com/thriftrw/thriftrw-go/gen/testdata/enums"
+	"github.com/thriftrw/thriftrw-go/gen/testdata/exceptions"
+	"github.com/thriftrw/thriftrw-go/gen/testdata/other_constants"
+	"github.com/thriftrw/thriftrw-go/gen/testdata/structs"
+	"github.com/thriftrw/thriftrw-go/gen/testdata/unions"
+)
+
+func _boolptr(v bool) *bool {
+	return &v
+}
+
+func _i64ptr(v int64) *int64 {
+	return &v
+}
+
+func _stringptr(v string) *string {
```

Comment:
unfortunately nope, no other way for literal values. we could move these into a `thriftrw` package though, rather than generating them with every package.